content/5_Plat/Geo_Pri.mdx

```diff
@@ -356,52 +409,80 @@ int main() {
 	cout << interior_points << ' ' << boundary_points;
 }
 ```
+
 </CPPSection>
 </LanguageSection>
 
 <Problems problems="standard" />
 
-## Resources
+# Angles
 
-<Resources>
-	<Resource source="CF" title="C++ - std::complex" url="22175" starred>
-		short description of operations
-	</Resource>
-	<Resource source="CPH" title="29 - Geometry" starred>
-		Complex #s, Points & Lines, Polygons, Distances
-	</Resource>
-	<Resource source="CF" title="Point Struct" url="48122" starred>
-		code, examples
-	</Resource>
-	<Resource
-		source="cp-algo"
-		title="Geometry - Elementary Operations"
-		url="https://cp-algorithms.com/"
-		starred
-	 />
-	<Resource source="CPC" title="12 - geometry" url="12_geometry" starred>
-		basics, polygon area, point in polygon
-	</Resource>
-	<Resource
-		source="CF"
-		title="vlecomte - Geometry Handbook"
-		url="59129"
-		starred
-	>
-		some material is quite advanced
-	</Resource>
-	<Resource source="CP2" title="7.2, 7.3 - Basic Geo, Polygons"/>
-</Resources>
+<FocusProblem problem="radians" />
 
-<!-- 	<Resource
-		source="TC"
-		title="Basic Geometry Concepts Pts 1,2"
-		url="geometry-concepts-basic-concepts"
-	>
-		broken code format
-	</Resource> -->
+## Explanation
 
-- [My Templates](<https://github.com/bqi343/USACO/tree/master/Implementations/content/geometry%20(13)/Primitives>)
+Working with angles only makes sense in the context of lines and segments. Therefore, let's consider the segments
+connecting the initial position and each dot: $(pos_x, pos_y)-(x, y)$. One such segment is inside the field of
+view if and only if it's slope is bigger than the lower bound and smaller than the upper bound. Since we're
+working with angles, we'll convert the slopes into radians.
+
```

Comment:
```suggestion
Working with angles only makes sense in the context of lines and segments. Therefore, let's consider the segments
connecting the initial position and each dot: $(pos_x, pos_y)-(x, y)$. One such segment is inside the field of
view if and only if its slope is bigger than the lower bound and smaller than the upper bound. Since we're
working with angles, we'll convert the slopes into radians.

```

Reply:
changed